src/Components/Components/src/LayoutAttribute.cs

```diff
@@ -36,5 +36,6 @@ public LayoutAttribute([DynamicallyAccessedMembers(Component)] Type layoutType)
     /// The type of the layout. The type must implement <see cref="IComponent"/>
     /// and must accept a parameter with the name 'Body'.
     /// </summary>
+    [DynamicallyAccessedMembers(Component)]
```

Comment:
Not wrong, but surprised that we need this now

Reply:
We do, otherwise we are getting:
```

Router.cs(164,13): error IL2074: value stored in field 'Microsoft.AspNetCore.Components.Routing.Router._notFoundLayoutType' does not satisfy 'DynamicallyAccessedMemberTypes.All' requirements. The return value of method 'Microsoft.AspNetCore.Components.LayoutAttribute.LayoutType.get' does not have matching annotations. The source value must declare at least the same requirements as those declared on the target location it is assigned to.

```